api/src/repository/AnswersRepository.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+class AnswersRepository extends Repository {
+
+    public function addAnswer($id, $answer) {
+
+        $stmt = $this->database->connect()->prepare('
+            INSERT INTO given_answers(id_question, given_answer) VALUES (?, ?)
+        ');
+
+        $stmt->execute([
+            $id,
+            $answer
+        ]);
+    }
+
+    public function getCountableAnswers($id): array {
+
+        $stmt = $this->database->connect()->prepare('
+            SELECT aa.answer, count(ga.id_given_answers) amount FROM available_answers aa
```

Comment:
Tutaj jest coś nie tak chyba, nie powinno być przypadkiem:
```suggestion
            SELECT ... amount FROM given_answers ga
            JOIN available_answers on ...
```